runelite-client/src/main/java/net/runelite/client/plugins/screenmarkers/ui/ScreenMarkerPanel.java

```diff
@@ -253,7 +253,7 @@ public void mouseExited(MouseEvent mouseEvent)
 		nameActions.add(cancel, BorderLayout.WEST);
 		nameActions.add(rename, BorderLayout.CENTER);
 
-		nameInput.setText(marker.getName());
+		nameInput.setText(marker.getDisplayName());
```

Comment:
Shouldn't this be `marker.getMarker().getName()` then?

Reply:
Yeah no need for new method, there is already getter for marker

Reply:
Ah, fixed